pkg/cmd/cmd.go

```diff
@@ -83,12 +94,41 @@ func newFullCoverageCommand() *cobra.Command {
 					float64(info.TotalCoveredLines)/float64(info.TotalLines)*100,
 				)
 			}
+
+			if kustoOption.UseKusto {
+				dbClient, err := dbclient.NewKustoClient(kustoOption)
+				if err != nil {
+					return fmt.Errorf("new kusto client: %w", err)
+				}
+
+				now := time.Now().UTC()
+				for _, info := range all {
+					err = dbClient.Store(context.Background(), &dbclient.Data{
+						PreciseTimestamp: now,
+						LinesCovered:     info.TotalCoveredLines,
+						LinesValid:       info.TotalLines,
+						ModulePath:       moduleHostPath,
+						FilePath:         info.Path,
+						Coverage:         float64(info.TotalCoveredLines) / float64(info.TotalLines),
```

Comment:
I put the preferred column names in this [issue](https://github.com/Azure/gocover/issues/12), please check it.

Reply:
It will be resolved in next PR. This PR focus on the kusto client.